apps/explorer/src/pages/transaction-result/programmable-transaction-view/Commands.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { type ProgrammableTransactionCommand } from '@mysten/sui.js';
+import clsx from 'clsx';
+
+import styles from '~/pages/transaction-result/TransactionResult.module.css';
+import { TableHeader } from '~/ui/TableHeader';
+
+interface Props {
+    commands: ProgrammableTransactionCommand[];
+}
+
+const formatCommandData = (commandData: any): string => {
+    if (Array.isArray(commandData)) {
+        return commandData
+            .map((data: any) => {
+                if (typeof data === 'object') {
+                    return JSON.stringify(data);
+                }
+                return data;
+            })
+            .join(', ');
+    }
+    return JSON.stringify(commandData);
+};
+
+export function Commands({ commands }: Props) {
+    if (!commands?.length) {
+        return null;
+    }
+
+    return (
+        <div data-testid="programmable-transactions-commands" className="mt-8">
+            <section
+                className={clsx([styles.txcomponent, styles.txgridcolspan2])}
+            >
+                <TableHeader>Commands</TableHeader>
+                <ul className="flex flex-col gap-8">
+                    {commands.map((command, index) => {
+                        const commandName = Object.keys(command)[0];
+                        const commandData: any =
```

Comment:
Should be a way to type this without resorting to Any.

Reply:
It will work without it, will remove